src/status_im2/contexts/wallet/common/utils.cljs

```diff
@@ -4,3 +4,27 @@
 (defn get-first-name
   [full-name]
   (first (string/split full-name #" ")))
+
+(defn calculate-raw-balance
+  [raw-balance decimals]
+  (if (not (js/isNaN (js/parseInt raw-balance)))
+    (/ (js/parseInt raw-balance) (Math/pow 10 (js/parseInt decimals)))
+    0))
```

Comment:
We call js/parseInt on the same value twice, We can extract it to a `let` and use it's value.

Reply:
👍 nice. Done